themes/hugo-creative-theme/layouts/partials/contact.html

```diff
@@ -0,0 +1,45 @@
+{{ "<!-- CONTACT -->" | safeHTML }}
+<section id="contact">
+    <div class="container">
+        <div class="row">
+            <div class="col-lg-8 col-lg-offset-2 text-center">
+                {{ with .Site.Params.contact.headline }}
+                    <h2 class="section-heading">{{ . }}</h2>
+                {{ end }}
+                {{ if and .Site.Params.contact.headline .Site.Params.contact.description }}
+                    <hr class="primary">
+                {{ end }}
+                {{ with .Site.Params.contact.description }}
+                    <p>{{ . | markdownify }}</p>
+                {{ end }}
+            </div>
+		</div>
+		<br>
+        <div class="row">
+            {{ if .Site.Params.contact.github }}
+                <div class="col-md-3 text-center">
+                    <i class="fab fa-github fa-3x wow bounceIn"></i>
+                    <p><a href="{{ .Site.Params.contact.github }}">GitHub</a></p>
+                </div>
+            {{ end }}
+            {{ if .Site.Params.contact.documentation}}
+                <div class="col-md-3 text-center">
+                    <i class="fas fa-book fa-3x wow bounceIn" data-wow-delay=".1s"></i>
+                    <p><a href="{{ .Site.Params.contact.documentation }}">Documentation</a></p>
+				</div>
+			{{ end }}
+            {{ if .Site.Params.contact.gitter }}
+                <div class="col-md-3 text-center">
+                    <i class="fab fa-gitter fa-3x wow bounceIn" data-wow-delay=".2s"></i>
+                    <p><a href="{{ .Site.Params.contact.gitter }}">Chat on Gitter</a></p>
+                </div>
+            {{ end }}
+            {{ if .Site.Params.contact.matrix }}
+                <div class="col-md-3 text-center">
+                    <i class="fas fa-comments fa-3x wow bounceIn" data-wow-delay=".2s"></i>
+                    <p><a href="{{ .Site.Params.contact.gitter }}">Chat on Matrix</a></p>
```

Comment:
Shouldn't here be `.Site.Params.contact.matrix` instead?

Reply:
Fixed